subsys/net/lib/lwm2m_client_utils/Kconfig

```diff
@@ -130,11 +145,58 @@ config LWM2M_CLIENT_UTILS_LOCATION_ASSIST_AGPS
 	  A-GPS assistance data is requested from lwm2m server and fed to the GNSS module
 	  after the data has been received.
 
+config LWM2M_CLIENT_UTILS_LOCATION_ASSIST_PGPS
+	bool "Use location based on P-GPS"
+	help
+	  P-GPS assistance data is requested from lwm2m server and fed to the GNSS module
+	  after the data has been received.
+
+config LWM2M_CLIENT_UTILS_LOCATION_ASSIST_PGPS_PREDICTION_COUNT
+	int "Number of predictions to be requested"
+	default 42
+	range 1 168
+	depends on LWM2M_CLIENT_UTILS_LOCATION_ASSIST_PGPS
+	help
+	  Default values give 6 predictions per day for a week. Depends on the starting day
+	  and the prediction interval.
+
+config LWM2M_CLIENT_UTILS_LOCATION_ASSIST_PGPS_PREDICTION_INTERVAL
+	int "Interval between predictions in minutes"
+	default 240
+	depends on LWM2M_CLIENT_UTILS_LOCATION_ASSIST_PGPS
+	help
+	  By default, 240 (4 hours) is used. Valid values are: 120, 240, 360, 480
```

Comment:
I would use _range 120 480_ here.

Reply:
Even though anything between not modulo 120 isn't supported?

Reply:
Yes.